Update placement.mdx (#25538)
hashicorp/nomad: website/content/docs/concepts/scheduling/placement.mdx

Refer to the [Node Pools][concept_np] concept page for more information.

## Understanding Evaluation Status

When a job cannot be immediately placed, Nomad creates a chain of evaluations
to manage placement. First, your initial evaluation may be marked complete even if 

> placement fails, then a blocked evaluation is created to retry placement when
> resources become available. Your job will remain pending till all allocations
> are placed.

Reviewer: ```suggestion
placement fails. Then Nomad creates a blocked evaluation to retry placement when
resources become available. Your job remains pending until Nomad places all allocations.
```

Suggestion to match our docs style guide. 
- We use [active voice](https://developers.google.com/style/voice) so I changed "is created" to "Nomad creates" and "all allocations
are placed" to "Nomad places all allocations."
- We use present tense so I changed "will remain" to "remains"

> For example, if your job has specific constraints that are not available. You
> will get an initial completed evaluation and a blocked evaluation which tracks
> ongoing placement attempts till a node that fits your constraints is available.
> 
> To troubleshoot placement issues, use `nomad eval status <eval_id>` and check
> for placement failures and linked evaluations in the output.

Reviewer: ```suggestion
For example, if your job has specific constraints that are not available, you
get an initial completed evaluation and a blocked evaluation. Nomad tracks
ongoing placement attempts until a node that fits your constraints is available.

To troubleshoot placement issues, use `nomad eval status <eval_id>`. Check the output 
for placement failures and linked evaluations.
```
